File: java/src/com/company/Node.java
package com.company;

public class Node {
    private String data;
    private String leftChild;
    private String rightChild;

    public Node(String data, String leftChild, String rightChild) {
        this.data = data;
        this.leftChild = leftChild;
        this.rightChild = rightChild;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getLeftChild() {
        return leftChild;
    }

    public void setLeftChild(String leftChild) {
        this.leftChild = leftChild;
    }

    public String getRightChild() {
        return rightChild;
    }

    public void setRightChild(String rightChild) {
        this.rightChild = rightChild;
    }

    @Override
    public String toString() {
        return "{" +
               "data:'" + data + '\'' +
               ", leftChild:'" + leftChild + '\'' +
               ", rightChild:'" + rightChild + '\'' +
               '}';
    }
}

File: c#/Node.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace binaryTree
{
    class Node
    {
        public String Data { get; set; }
        public String LeftChild { get; set; }
        public String RightChild { get; set; }

        public Node(String data, String leftChild, String rightChild)
        {
            this.Data = data;
            this.LeftChild = leftChild;
            this.RightChild = rightChild;
        }

        public override string ToString()
        {
            return "{" + "data:'" + Data + "', " + "leftChild:'" + LeftChild + "', " + "rightChild:'" + RightChild + "'}";
        }
    }
}
